Formula/delly.rb

```diff
@@ -0,0 +1,27 @@
+class Delly < Formula
+  # cite Rausch_2012: "https://doi.org/10.1093/bioinformatics/bts378"
+  desc "Structural variant discovery by paired-end and split-read analysis"
+  homepage "https://github.com/tobiasrausch/delly"
+  url "https://github.com/tobiasrausch/delly.git",
+      :tag      => "v0.8.3",
+      :revision => "56ab2900da269568dc327045913ae77ff3919df1"
+
+
```

Comment:
👇Is this magic, or what! Thank you, @jonchang!

Reply:
Very nice !